### ISSUE TITLE
mGBA on the wii doesn't have the Color correction feature?

### ISSUE THREAD

**Issue by Ledoeye** [mgba-emu/mgba#2666]
Hi! I been playing some gba games on the Wii and although games look amazing specially when played on a CRT display(240p) i can't deny that i miss the feature Color Correction from the PC mGBA build. Is there any way to get the feature Color correction on the Wii? Thanks.

**Comment by Bitwolfies** [comment]
Wait, the PC version has it? I thought it was in Retroarch only, I cannot for the life of me figure out how to enable or disable it in standalone

**Comment by Ledoeye** [comment]
Yeah apparently it's in Retroarch only, my bad. I absolutely love how GBA games look with the original hardware colors.
https://imgur.com/a/X1u8Xyd
This is how i usually play my GBA games on my PC CRT, it looks great IMO. I would love this color correction feature on the Wii tho, maybe one day :D

**Comment by endrift** [comment]
I would like to add a CLUT pass to mGBA Wii someday that can do this, yes.

**Comment by Ledoeye** [comment]
Yes! Thank you so much for all the hard work! @endrift
